rex.utilities.exceptions.RetryError
===================================

.. currentmodule:: rex.utilities.exceptions

.. autoexception:: RetryError